project/Dependencies.scala

```diff
@@ -293,14 +293,17 @@ object Dependencies {
         jacksonCore,
         jacksonAnnotations,
         jacksonDatabind,
-        jacksonScala,
         jacksonJdk8,
         jacksonJsr310,
         jacksonParameterNames,
         jacksonCbor,
         lz4Java,
         Test.junit,
-        Test.scalatest)
+        Test.scalatest) ++
+    (if (getScalaVersion() == scala3Version)
+      Seq("com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.13.0-rc1" % "test")
+    else
+      Seq(jacksonScala))
```

Comment:
I think it will be better if we use:

```
val jacksonScala = ("com.fasterxml.jackson.module" %% "jackson-module-scala" % jacksonVersion).cross(CrossVersion.for3Use2_13) 
```

because we don't want to touch the jackson version.

Reply:
That would make `jackson-module-scala_2.13` artifact a transitive dependency of our `akka-serialization-jackson_3` artifact, which I'd say we want to avoid.

This is only touching the Jackson version in test scope, so that might be fine? (with the cross-built artifact many more tests fail)

Reply:
Ok, let's go for this solution, but...

> hat would make jackson-module-scala_2.13 artifact a transitive dependency of our akka-serialization-jackson_3 artifact, which I'd say we want to avoid.

I don't think we should want to avoid this. My understanding is that this should work and it's preferable then upgrading the jackson version. 

Anyway, as you said, that's only a test dependency. So fine if it works using 2.13.0-rc1.

Reply:
> > That would make jackson-module-scala_2.13 artifact a transitive dependency of our akka-serialization-jackson_3 artifact, which I'd say we want to avoid.
> 
> My understanding is that this should work

Not necessarily: it *might* work for *some* libraries that don't require changes to work between Scala 2.13 and Scala 3, but jackson-module-scala did need changes for scala 3 (i.e. https://github.com/FasterXML/jackson-module-scala/issues/479 and similar issues), so bringing in the known-not-to-work Scala 2.13 artifact isn't too helpful in this particular case.

Reply:
(https://scala-lang.org/blog/2021/04/08/scala-3-in-sbt.html#using-scala-213-libraries-in-scala-3 also recommends against using `cross` for transitive dependencies)